src/platform/apollolake/include/platform/asm_memory_management.h

```diff
@@ -53,7 +53,7 @@
 	// SEGMENT #0
 	movi \az, (SHIM_BASE + SHIM_HSPGISTS)
 	movi \ax, (SHIM_BASE + SHIM_HSPGCTL)
-	movi \ay, HPSRAM_MASK()
+	movi \ay, HPSRAM_MASK(0)
 	s32i \ay, \ax, 0
```

Comment:
This change looks totally unrelated to the commit message. Can you make this a separate patch.

Reply:
It is related  since the  macro was moved to cavs/memory.h file and needs a parameter. This parameter is integral to the macro move.(it won't build if it is not changed)